recipes/dandischema/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "dandischema" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 2a1757510b811e49a392da08d96d397ab3137dbab2cd5e09905659ffea695491
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+    - setuptools
+  run:
+    - python >=3.7
+    - jsonschema
+    - pydantic[email] >=1.8.1
+    - ruamel.yaml
+    - typing_extensions
```

Comment:
curious: typing_extenions is needed ONLY for py37, so why the suggestion by @dopplershift (IIRC) which @jwodder followed? i.e. does py38 and so on somehow "provide" (in Debian terms) that package or it would be needlessly installed now even whenever not needed?